src/server/schema.js

```diff
@@ -42,60 +47,155 @@ const gqlTypeToAggsHistogramName = {
 
 const getAggsHistogramName = (gqlType) => {
   if (!gqlTypeToAggsHistogramName[gqlType]) {
-    throw new Error(`Invalid elasticsearch type ${gqlType}`);
+    // throw new Error(`Invalid elasticsearch type ${gqlType}`);
+    return '';
```

Comment:
why are we not throwing the error here?

Reply:
because we have a nested (object) type like this:

```
ActionableMutations(
ActiveClinicalTrials: [String]
BxMorphology: [String]
BxType: [String]
Chicago_ID: [String]
ClinicallyActionable: [String]
Consequence: [String]
DaysFromAnchorDateToBxDate: [Float]
DaysFromAnchorDateToBxResultsDate: [Float]
DrugsAssociatedWithResistance_IndicatedTumorTypes: [Float]
DrugsAssociatedWithSensitivity_IndicatedTumorTypes: [String]
DrugsAssociatedWithSensitivity_OtherTumorTypes: [String]
Exon_PGDonly: [Float]
Fold: [Float]
Gene: [String]
Lab: [String]
MutantFraction: [Float]
Mutation: [String]
TotalNumberMutations: [Float]
TrialPhase: [Float]
): [ActionableMutations]
```

In patients, there could be a list of ActionableMutations

Reply:
then can you dynamically add `ActionableMutations` key to `gqlTypeToAggsHistogramName` object? or maybe just check `gqlType` is valid will also make sense?

Reply:
how do we determine a valid gqlType. For Histogram, I don't know if adding it more Object type create any issue

Reply:
> then can you dynamically add `ActionableMutations` key to `gqlTypeToAggsHistogramName` object? or maybe just check `gqlType` is valid will also make sense?

`ActionableMutations` is not harded code type. We must consider the way to add field dynamically. It is unnecessary anyway